### DISCUSSION TITLE
show tooltip Always without trigger

### DISCUSSION THREAD

**Discussion by akshaynarisetti** [rsuite/rsuite#2705] [Q&A] [answered]
How to show a tooltip always, for example when some state is true (Irrespective of whisper trigger)

any response is appreciated @simonguo

**Comment by simonguo** [comment, marked as answer]
Use `open` to force the Tooltip to appear.

```tsx
<Whisper speaker={<Tooltip> Tooltip!</Tooltip>} open>
  <Button> button </Button>
</Whisper>
```
